qiskit/quantum_info/operators/symplectic/clifford.py

```diff
@@ -368,6 +442,11 @@ def to_circuit(self):
                Phys. Rev. A 70, 052328 (2004).
                `arXiv:quant-ph/0406196 <https://arxiv.org/abs/quant-ph/0406196>`_
         """
+        # pylint: disable=cyclic-import
+        from qiskit.quantum_info.synthesis.clifford_decompose import (
+            decompose_clifford,
+        )
+
         return decompose_clifford(self)
```

Comment:
I guess this should be updated at some point to allow directly adding Clifford op to circuit without decomposing, and have decomposition be called in that Instructions `definition` for unrolling.

Reply:
Indeed, we are working on it as part of a more general synthesis framework.

Reply:
Great. It should be resolved in other pull requests.